usr/sbin/p11sak/p11sak.c

```diff
@@ -2550,6 +2775,49 @@ static CK_RV close_session(CK_SESSION_HANDLE session)
     return rc;
 }
 
+static CK_RV parse_file() {
+    CK_RV rc = CKR_OK;
+    FILE *fp;
+    char *file_loc = getenv("P11SAK_DEFAULT_CONF_FILE");
+
+    // open and parse file
+    if ((fp = fopen(file_loc, "r")) == NULL) {
+        if (file_loc) {
+            fprintf(stderr, "Failed to load %s: %s\n", file_loc, strerror(errno));
+        }
+
+        char *buf;
+        struct passwd *pw;
```

Comment:
Variable definitions should be only at the beginning of a code block { ... }.

Reply:
Done